Makefile

```diff
@@ -267,69 +257,53 @@ pylint:
-	${DOCKER} tag services_director:latest itisfoundation/director:cache
-	${DOCKER} tag services_storage:latest itisfoundation/storage:cache
-	${DOCKER} tag services_webclient:build itisfoundation/webclient:cache
-	${DOCKER} tag services_webserver:latest itisfoundation/webserver:cache
-
-	${DOCKER} push itisfoundation/apihub:cache
-	${DOCKER} push itisfoundation/director:cache
-	${DOCKER} push itisfoundation/sidecar:cache
-	${DOCKER} push itisfoundation/storage:cache
-	${DOCKER} push itisfoundation/webclient:cache
-	${DOCKER} push itisfoundation/webserver:cache
-
-TRAVIS_PLATFORM_STAGE_VERSION=staging-$(shell date +"%Y-%m-%d").${TRAVIS_BUILD_NUMBER}.$(shell git rev-parse HEAD)
-TRAVIS_PLATFORM_STAGE_LATEST=staging-latest
-travis-push-staging-images:
-	${DOCKER} tag services_apihub:latest itisfoundation/apihub:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_webserver:latest itisfoundation/webserver:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_sidecar:latest itisfoundation/sidecar:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_director:latest itisfoundation/director:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_storage:latest itisfoundation/storage:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} tag services_apihub:latest itisfoundation/apihub:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} tag services_webserver:latest itisfoundation/webserver:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} tag services_sidecar:latest itisfoundation/sidecar:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} tag services_director:latest itisfoundation/director:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} tag services_storage:latest itisfoundation/storage:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/apihub:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/webserver:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/sidecar:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/director:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/storage:${TRAVIS_PLATFORM_STAGE_VERSION}
-	${DOCKER} push itisfoundation/apihub:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/webserver:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/sidecar:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/director:${TRAVIS_PLATFORM_STAGE_LATEST}
-	${DOCKER} push itisfoundation/storage:${TRAVIS_PLATFORM_STAGE_LATEST}
+.PHONY: push-cache	
+push-cache:
+	${DOCKER_COMPOSE} -f services/docker-compose.yml -f services/docker-compose.cache.yml push ${CACHED_SERVICES_LIST}
+
+
+# staging ----------------
+.PHONY: build-staging
+build-staging:
+	export DOCKER_IMAGE_PREFIX=itisfoundation/; \
+	export DOCKER_IMAGE_TAG=staging-latest; \
+	${MAKE} build
+
+TRAVIS_PLATFORM_STAGE_VERSION := staging-$(shell date +"%Y-%m-%d").${TRAVIS_BUILD_NUMBER}.$(shell git rev-parse HEAD)
+.PHONY: push-staging
+push-staging:
+	export DOCKER_IMAGE_PREFIX=itisfoundation/ \
```

Comment:
I would rather call ``DOCKER_IMAGE_PREFIX`` as
``DOCKER_IMAGE_REPO `` such that ``$DOCKER_IMAGE_REPO/$DOCKER_IMAGE_APP:$DOCKER_IMAGE_TAG``

See https://docs.docker.com/engine/reference/commandline/tag/